pkg/kudoctl/files/file.go

```diff
@@ -0,0 +1,64 @@
+package files
+
+import (
+	"crypto/sha256"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
+)
+
+// CopyOperatorToFs used with afero usually for tests to copy files into a filesystem.
+// copy from local file system into in mem
+func CopyOperatorToFs(fs afero.Fs, opath string, base string) {
```

Comment:
This method looks like a general-purpose file method but it doesn't handle recursive directories, symlinks and what not. Wdyt about "steeling" this method from [docker](https://github.com/moby/moby/blob/8e610b2b55bfd1bfa9436ab110d311f5e8a74dcb/daemon/graphdriver/copy/copy.go)?  Or make it test-private as Alena suggested.

Reply:
I'm not sure about stealing... I'll look.  It was meant to be enough to run the tests needed.  I already have tests for future PR needing it... hence not private.